### ISSUE TITLE
Add short, character limited description field to schedule

### ISSUE THREAD

**Issue by marrold** [emfcamp/Website#1156]
Some of the descriptions in the schedule are incredibly long and detailed which makes displaying them challenging on space limited mediums (Badge, Teletext, etc)

It would be great if there was additional field in the schedule for a shorter description, ideally characted limited.

This is one for next time given that everyone's already submitted their proposals and the scheduling is underway.

Thanks!

**Comment by marrold** [comment]
Someones started working on a Dial Up BBS service to display the schedule and they've hit the same issue with the large descriptions in the sample data [here](https://www.emfcamp.org/schedule/2022.json)

**Comment by russss** [comment]
I think the question here is: who writes these short descriptions and when? We probably don't want to ask for them up-front on the CfP, as that raises the bar to entry even more.

Do we ask submitters to write them at finalisation, or do we write them ourselves?

**Comment by jayaddison** [comment]
@marrold can you share a preview of what the schedule would look like in such a view?

Initially the existing `title` fields appear short and succinct to me -- but in some cases I suppose they do not make the content of the event entirely clear.

I'm wondering if it's only cases where all of the following are true could be problematic:

  * Title is opaque (which can be fine; we should allow for things like fun)
  * Description is long
  * Description's first sentence doesn't immediately clarify the content

**Comment by marrold** [comment]
@jayaddison I'm not really proposing any UI changes to the schedule, just the underlying data / API, for consumption from other things.

Here's an example of a very verbose description that's hard to display on anything with limited screen size, via text to speech etc - https://www.emfcamp.org/schedule/2024/161-gas-boilers-suck-hack-yours-today-save-money-save-the-planet

The description could be summarised as "Controlling a gas boiler with a Raspberry Pi" - Slightly mundane for a title, but a reasonable description to read on a badge.

**Comment by jayaddison** [comment]
@marrold OK, gotcha.  Do you expect that these short summaries would generally be surfaced alongside the title?

(from a speaker/event-organiser point-of-view, I'd want and expect my event(s) to be redistributed using their original titles)

**Comment by jayaddison** [comment]
@marrold Yep, that's totally fine.  If it's only a small percentange of events where the `title` / truncated `description` are not suitable (5%?  20%?) -- I could volunteer to write some short descriptors and offer them as a JSON file?

They wouldn't be EMF-official descriptors, but I enjoy writing myself short summarizations of films before attending film festivals.. this doesn't seem so different.

**Comment by marksteward** [comment]
There's a short_description column since the refactor (#1868) on Schedule Items, that should be exposed on the feeds when they're turned on.
